### PR TITLE
create a aboutme.md file inside aboutme folder containing my resume

### PR DESCRIPTION
**Pull request by JamalGuyo**
- created a folder called aboutme
- inserted an aboutme.md file
- aboutme.md file contains my resume

### CODE REVIEW

File: aboutMe/AboutMe.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+[TOC]
+
+---
+
+## Basic information
+
+### Name
+
+Jamal Guyo
```
Comment (marvinhosea) [inline aboutMe/AboutMe.md]: You can put the name inline to the name title  like this  
 **Name:**  Jamal Guyo

File: aboutMe/AboutMe.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+### Title
+
+Software Developer
```
Comment (marvinhosea) [inline aboutMe/AboutMe.md]: You can put the title inline with the title above 
**Title:** Software developer

**Review by marvinhosea — APPROVED** [review]
@JamalGuyo good job. You followed the assignment requirement and also used git very well. We appreciate this effort, keep it up.

**Comment by marvinhosea** [comment]
Feel free to merge this pull request without resolving the above issues or after resolving the above issues

File: aboutMe/AboutMe.md (hunk 1 above)
Comment (JamalGuyo) [reply aboutMe/AboutMe.md]: resolved

File: aboutMe/AboutMe.md (hunk 2 above)
Comment (JamalGuyo) [reply aboutMe/AboutMe.md]: resolved

**Pull request merged**